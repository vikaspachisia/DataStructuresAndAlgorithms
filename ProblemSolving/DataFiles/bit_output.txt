#1 1
5
3
#2 2
6
9
#3 3
5 3 4
#4 4
12
#5 5
5
6
#6 6
6
12
#7 7
5 6 7
#8 8
18
