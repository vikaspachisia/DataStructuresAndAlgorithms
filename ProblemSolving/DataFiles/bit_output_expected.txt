#1
5
3
#2
6
9
#3
5 3 4
#4
12
#5
5
6
#6
6
12
#7
5 6 7
#8
18